src/fragments/lib/storage/android/list.mdx

```diff
@@ -71,7 +70,209 @@ StoragePagedListOptions options = StoragePagedListOptions.builder()
+
+```bash
+Path: public/photos/photo1.jpg
+Path: public/photos/vacation/photo1.jpg
+Path: public/photos/thumbnails/photo1.jpg
+```
+
+In order to exclude objects within the `vacation` and `thumbnails` subpaths, you can set the `subpathStrategy` option to `SubpathStrategy.Exclude()`:
+
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+StoragePagedListOptions options = StoragePagedListOptions.builder()
+    .setPageSize(1000)
+    .setSubpathStrategy(SubpathStrategy.Exclude())
+    .build();
+
+Amplify.Storage.list(
+    StoragePath.fromString("public/photos/"),
+    options,
+    result -> {
+        for (StorageItem item : result.getItems()) {
+            Log.i("MyAmplifyApp", "Item: " + item.getPath());
+        }
+        Log.i("MyAmplifyApp", "Next Token: " + result.getNextToken());
+    },
+    error -> Log.e("MyAmplifyApp", "List failure", error);
+);
+```
+
+</Block>
+<Block name="Kotlin - Callbacks">
+
+```kotlin
+val options = StoragePagedListOptions.builder()
+    .setPageSize(1000)
+    .setSubpathStrategy(SubpathStrategy.Exclude())
+    .build()
+
+Amplify.Storage.list(StoragePath.fromString("public/photos/"), options,
+    { result ->
+        result.items.forEach { item ->
+            Log.i("MyAmplifyApp", "Item: ${item.path}")
+        }
+        Log.i("MyAmplifyApp", "Next Token: ${result.nextToken}")
+    },
+    { Log.e("MyAmplifyApp", "List failure", it) }
+)
+```
+
+</Block>
+<Block name="Kotlin - Coroutines">
+
+```kotlin
+val options = StoragePagedListOptions.builder()
+    .setPageSize(1000)
+    .setSubpathStrategy(SubpathStrategy.Exclude())
+    .build()
+
+try {
+    val result = Amplify.Storage.list(StoragePath.fromString("public/photos/"), options)
+    result.items.forEach {
+        Log.i("MyAmplifyApp", "Item: $it")
+    }
+    Log.i("MyAmplifyApp", "next token: ${result.nextToken}")
+} catch (error: StorageException) {
+    Log.e("MyAmplifyApp", "List failure", error)
+}
+```
+
+</Block>
+<Block name="RxJava">
+
+```java
+StoragePagedListOptions options = StoragePagedListOptions.builder()
+        .setPageSize(1000)
+        .setSubpathStrategy(SubpathStrategy.Exclude())
+        .build();
+
+RxAmplify.Storage.list(StoragePath.fromString("public/photos/"), options)
+        .subscribe(
+            result -> {
+                for (StorageItem item : result.getItems()) {
+                    Log.i("MyAmplifyApp", "Item: " + item.getPath());
+                }
+                Log.i("MyAmplifyApp", "Next Token: " + result.getNextToken());
+            },
+            error -> Log.e("MyAmplifyApp", "List failure", error);
+        );
+```
+
+</Block>
+</BlockSwitcher>
+
+The response will only include objects within the `public/photos/` path and will also provide a list of the excluded subpaths:
+
+```bash
+Path: public/photos/photo01.jpg
+Subpath: public/photos/vacation/
+Subpath: public/photos/thumbnails/
+```
+
+The default delimiter character is `"/"`, but this can be changed by supplying a custom delimiter:
```

Comment:
i'm not sure I fully understand the use of a different delimiter than "/". Can you provide a similar section below the code snippets to show the response? For example,
```
The response will only include objects ... and also provide a list of the excluded subpaths:


Path: public/photos-photo01.jpg
subpath: public/photos-vacation/
```

Reply:
I've added a sentence to clarify the expected response.  I'm not including the sample example paths, since there isn't context for the sample paths to make sense.